### PR TITLE
fix: (swap): Slippage error in confirm modal doesn't have settings and docs link in other languages

### PR DESCRIPTION
**Pull request by memoyil**
This pr separates useSwapCallback from translation function

Issue: 

image

image

### CODE REVIEW

File: src/views/Swap/components/SwapCommitButton.tsx [hunk 1]
```diff
@@ -116,7 +117,7 @@ export default function SwapCommitButton({
         setSwapState({
           attemptingTxn: false,
           tradeToConfirm,
-          swapErrorMessage: error.message,
+          swapErrorMessage: getErrorReasonFromTransactionError(error),
```
Comment (chef-jojo) [inline src/views/Swap/components/SwapCommitButton.tsx:120]: can you help check if UserRejectedRequestError
I think we shouldn't reset the whole swap state there

Comment (chef-jojo) [reply src/views/Swap/components/SwapCommitButton.tsx:120]: Ah right. I mean we don't need to show transaction reject?
Just revert to no tx attempting

File: apps/web/src/hooks/useSwapCallback.ts [hunk 2]
```diff
@@ -188,17 +184,14 @@ export function useSwapCallback(
             return response.hash
           })
           .catch((error: any) => {
-            // if the user rejected the tx, pass this along
-            if (error?.code === 4001) {
-              throw new Error('Transaction rejected.')
```
Comment (chefilip) [inline apps/web/src/hooks/useSwapCallback.ts:193]: should we keep this?

Comment (chefilip) [reply apps/web/src/hooks/useSwapCallback.ts:193]: Got it.

**Review by chefilip — DISMISSED** [review]

File: apps/web/src/hooks/useSwapCallback.ts (hunk 2 above)
Comment (chef-jojo) [reply apps/web/src/hooks/useSwapCallback.ts:192]: @memoyil Let's do the same thing for farms. I think it's still popup the error when user reject

**Pull request closed without merging**